site/_components/fifty-fifty.njk

```diff
@@ -0,0 +1,38 @@
+<!--{#
+  Renders a Fifty Fifty module.
+
+  @param {object} content - An object containing the following attributes.
+  @param {string} content.title - title of the content.
+  @param {string} content.copy - Body copy of the content.
+
+  @param {object} cta (optional) - An object containing the following attributes.
+  @param {string} cta.url - URL for the cta.
+  @param {string} cta.text - CTA link text.
+
+  @param {object[]} images - An array of image objects containing the following attributes.
+  @param {string} images[].src - Image source.
+  @param {string} images[].alt - Image description.
+
+  @param {object} modifiers (optional) - An object containing the following attributes.
+  @param {string} modifiers.class A custom class.
+  @param {boolean} modifiers.flip - Invert the order of the image and text content.
+#}-->
+{% from 'icon.njk' import icon %} {% macro fiftyFifty(content, cta, images, modifiers) %}
+<article class="fifty-fifty {{modifiers.class}} {{ 'fifty-fifty--flip' if modifiers.flip }}">
+  {% set imageWrapper %}
+  <div class="fifty-fifty__image-wrapper">
+    {% for image in images %}
+    <img src="{{image.src}}" alt="{{image.alt}}" class="fifty-fifty__image" />
+    {% endfor %}
+  </div>
+  {% endset %} {% if modifiers.flip %}{{ imageWrapper | safe }}{% endif %}
```

Comment:
This is cool! I didn't know you could use `set` for a block of HTML! I bet we could clean up a bunch of our code with this! My only recommendation here is to comment that as I didn't know what it was looking at it to begin with and I bet others won't either.
```suggestion
  <!--{# Set imageWrapper to provided HTML #}-->
  {% set imageWrapper %}
  <div class="fifty-fifty__image-wrapper">
    {% for image in images %}
    <img src="{{image.src}}" alt="{{image.alt}}" class="fifty-fifty__image" />
    {% endfor %}
  </div>
  {% endset %} {% if modifiers.flip %}{{ imageWrapper | safe }}{% endif %}
```